pkg/ansible/proxy/proxy.go

```diff
@@ -66,6 +66,8 @@ func CacheResponseHandler(h http.Handler, informerCache cache.Cache, restMapper
 				log.Error(err, "Failed to convert request")
 				break
 			}
+			// Set Content-Type header
+			w.Header().Set("Content-Type", "application/json")
```

Comment:
Let's do this closer to the other cache header we set. I don't think this would be an issue but don't want to set this, if the api-server is not going to send json back.